docs/checks/internal/checks.yaml

```diff
@@ -630,6 +630,8 @@ checks:
       If a signature is found in the assets for each release, a score of 8 is given.
       If a [SLSA provenance file](https://slsa.dev/spec/v0.1/index) is found in the assets for each release (*.intoto.jsonl), the maximum score of 10 is given.
 
+      This check looks for the most recent releases associated with an artifact. It ignores the source code-only releases that are created automatically by GitHub.
```

Comment:
there may be some confusion about what the most recent means. I think it's 30 for github by defualt?

Reply:
Good point, it's better to specify the number of days

Reply:
I don't think it's related to the number of days currently.